T2-TECHNICAL-OVERVIEW.md

```diff
@@ -0,0 +1,137 @@
+    
+### Device Related Code
+#####[Tessel 2 Operation System (OpenWRT)](https://www.github.com/tessel/openwrt-tessel)
+The primary processpr of the Tessel 2 runs a very lightweight version of Linux called OpenWRT. OpenWRT provides all of the TCP/IP drivers, threading/schedule support, and runs Node, Rust or whatever other language you're using with Tessel 2. This repo contains all of the source files of the Tessel build of OpenWRT.
+
+#####[Tessel 2 Co-processor Firmware](https://www.github.com/tessel/t2-firmware)
+Tessel 2 features a co-processor that manages a handful of responsibilities like routing USB communication from the CLI and the realtime operations on the GPIO pins of the two module ports. This repo contains not only the firmware that drives this microcontroller but hardware-related scripts that define the interface to other languages (like [tessel.js](https://www.github.com/tessel/t2-firmware/node/tessel.js) for Node scripts) so that the available hardware functionality doesn't get out of sync with what is exposed to the OpenWRT processor.
+
+### Hardware Designs
+#####[Tessel 2 Hardware Design Files](https://www.github.com/tessel/t2-hardware)
+These are the actual schematic and assembly files for the Tessel 2 hardware. You can find information about the parts that are used and how they are all connected to each other. These files were created with [KiCad](http://www.kicad-pcb.org/display/KICAD/KiCad+EDA+Software+Suite), a completely open source electronics design tool.
+#####[Tessel Parts Library](https://github.com/tessel/tm-kicad-library)
+This repo contains all of the KiCad part models for all Tessel hardware (not just Tessel 2).
+
+### Modules
+
+##### 10-Pin Modules
+There are 9 compatible 10-pin hardware modules for Tessel 2 available at the time of this writing:
+* [Ambient (Light & Sound) Module](https://github.com/tessel/ambient-attx4)
+* [RFID/NFC Module](https://github.com/tessel/rfid-pn532)
+* [Accelerometer Module](https://github.com/tessel/accel-attx4)
+* [Bluetooth Low Energy Module](https://github.com/tessel/ble-ble113a)
+* [Climate (temperature & humidity) Module](https://github.com/tessel/climate-si7020)
+* [GPS Module](https://github.com/tessel/gps-a2235h)
+* [Infrared Module](https://github.com/tessel/ir-attx4)
+* [Relay Module](https://github.com/tessel/relay-mono)
+* [Servo Module](https://github.com/tessel/servo-pca9685)
+
+##### USB Modules
+USB Modules haven't been released yet so some don't have Github repos and many are in progress. In general, Tessel supports the following kind of USB modules which you can find from many online vendors:
```

Comment:
These are already supported? You mention that many are in progress; where can I see the in-progress ones?